.junie/guidelines.md

```
@@ -0,0 +1,12 @@
+# Project Guidelines
+    
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 4

(MD009, no-trailing-spaces)